packages/design-system-react/src/components/button-base/ButtonBase.tsx

```diff
@@ -83,8 +83,7 @@ export const ButtonBase = React.forwardRef<HTMLButtonElement, ButtonBaseProps>(
       'inline-flex items-center justify-center',
       'rounded-full px-4',
       'text-default font-medium',
-      'transition-colors duration-150',
-      'bg-muted hover:bg-muted-hover active:bg-muted-pressed',
+      'bg-muted',
```

Comment:
Removing transition from `ButtonBase` as we want to apply specific transitions and motions to individual variants